### ISSUE TITLE
/etc/cron.hourly/enable_secalerts cronjob fails with errno 6

### ISSUE THREAD

**Issue by clst** [turnkeylinux/tracker#708]
If you specify a valid email address containing a "+" sign, the curl call returns
`Request.MalformedArgument:Argument not recognized/supported: email`
and fails.

**Comment by JedMeister** [comment]
Hi, thanks for your bug report. 

FWIW, on quick review, I thought that this was a duplicate of #155 (which was resolved in v14.1).

After re-reading, it seems I was jumping to conclusions!

**Comment by clst** [comment]
Yes, to clarify: I think the issue is the server-side script hub.turnkeylinux.org/api/server/secalerts not the curl call.

**Comment by JedMeister** [comment]
Thanks for the clarification. I'll adjust the tags; as you say it seems to be related to the Hub API rather than the other scripts.
